RecoEgamma/EgammaIsolationAlgos/interface/EgammaL1TkIsolation.h

```diff
@@ -0,0 +1,58 @@
+#ifndef RECOEGAMMA_EGAMMAISOLATIONALGOS_EGAMMAL1TKISOLATION_H
+#define RECOEGAMMA_EGAMMAISOLATIONALGOS_EGAMMAL1TKISOLATION_H
+
+#include "DataFormats/L1TrackTrigger/interface/L1Track.h"
+#include "DataFormats/TrackReco/interface/TrackBase.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+
+//author S. Harper (RAL/CERN)
+//based on the work of Swagata Mukherjee and Giulia Sorrentino
+
+class EgammaL1TkIsolation {
+public:
```

Comment:
CMS code rule 4.25 (http://cms-sw.github.io/cms_coding_rules.html): "Each class may have only one each of public, protected, and private sections, which must be declared in that order."

Reply:
done

Reply:
to be pedantic, per 4.26, the order of functions in the .h and .cc should also be the same